README.rst

```diff
@@ -25,6 +33,8 @@ sudo apt-get update && sudo apt-get install python-jujuclient python-txjuju
 Building
 ---------
 
-Generally speaking, this package is built and delivered to PPAs from a recipe here: https://code.launchpad.net/fake-juju/+recipes.
+Generally speaking, this package is built and delivered to PPAs from a recipe_.
 
 You can build it locally running 'dpkg-buildpackage -b'.
```

Comment:
Let's make this a string literal preceded by :: and indented to render command line literals in rst.


s/running /running::
  $ dpkg-buildpackage -b/


We also need to fix the sudo apt lines below with a preceding literal declaration.
We just are missing a trailing ":" and indenting sudo apt* lines.

s/can be added with:/can be added with::/